src/KeyVault/KeyVault/Commands/Key/GetAzKeyVaultRandomNumber.cs

```diff
@@ -10,7 +10,7 @@
 namespace Microsoft.Azure.Commands.KeyVault.Commands.Key
 {
     [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzurePrefix + "KeyVaultRandomNumber", DefaultParameterSetName = GetByHsmNameParameterSet)]
-    [OutputType(typeof(string))]
+    [OutputType(typeof(string), typeof(byte[]))]
```

Comment:
```suggestion
    [OutputType(typeof(string), typeof(byte))]
```
See https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/design-guidelines/cmdlet-best-practices.md#output-type